src/common/components/select-repository-list/index.js

```diff
@@ -152,6 +166,14 @@ export class SelectRepositoryListComponent extends Component {
 
     return (
       <div>
+        <div className={ styles.repoSearch }>
```

Comment:
As search input is part of heading of this dialog it shouldn't be rendered here and positioned absolutely.

We need to put the `SearchInput` where it belongs (in `SearchRepositoriesPage` component) and only pass `searchText` or something like this as a property to `SearchRepositoriesList`.